gitlab/v4/objects/merge_requests.py

```diff
@@ -327,6 +327,24 @@ def rebase(self, **kwargs: Any) -> Union[Dict[str, Any], requests.Response]:
         data: Dict[str, Any] = {}
         return self.manager.gitlab.http_put(path, post_data=data, **kwargs)
 
+    @cli.register_custom_action("ProjectMergeRequest")
+    @exc.on_http_error(exc.GitlabMRResetApprovalError)
+    def reset_approvals(
+        self, **kwargs: Any
+    ) -> Union[Dict[str, Any], requests.Response]:
+        """Attempt to reset approvals onto the target Merge Request
+
+        Args:
+            **kwargs: Extra options to send to the server (e.g. sudo)
+
+        Raises:
+            GitlabAuthenticationError: If authentication is not correct
+            GitlabMRResetApprovalError: If reset approval failed
+        """
+        path = f"{self.manager.path}/{self.encoded_id}/reset_approvals"
+        data: Dict[str, Any] = {}
```

Comment:
I don't think we necessarily to supply the empty data here, but I see we do the same for rebase above, so let's leave it.

Reply:
I'm thinking that maybe that is there, so we can `skip_ci` to rebase as data? https://docs.gitlab.com/ee/api/merge_requests.html#rebase-a-merge-request

If that makes sense, we could remove it from reset_approvalls, OTOH it doesn't really affect the functionality of reset_approvals, and it can be ready if reset_approvalls is expanded to, lets say, reset a single approval, wdyt?

Reply:
Oh, that wouldn't actually work as we just define an empty dict in the method, but nice find! Unrelated, but we should also support that in rebase :) That's also a recent feature IIRC.